src/lib/modernExtend.ts

```diff
@@ -2686,4 +2691,17 @@ export function bindCluster(args: {cluster: string | number; clusterType: "input
     return {configure, isModernExtend: true};
 }
 
+export function reportAttribute(args: {
```

Comment:
I don't think this is used? Please remove

Reply:
currently not used, but can be used along with/like bindCluster modernExtend https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/lib/modernExtend.ts#L2684C17

Reply:
If it is not used than I propose to remove it for now.